react/react1/week1/todo-tracker/src/App.js

```diff
@@ -0,0 +1,26 @@
+import "./App.css";
+import React from "react";
+import ToDoListComp from "./components/ToDoList";
+import toDoList from "./data/toDoList";
+import SearchComp from "./components/Search";
+
+function App() {
+  const [filterToDoList, setFilterToDoList] = React.useState(toDoList);
+
+  return (
+    <div className="App">
+      <SearchComp
+        onFilteredToDoListChange={(text) =>
+          setFilterToDoList(
+            toDoList.filter((tododata) =>
+              tododata.description.toLowerCase().includes(text.toLowerCase())
+            )
+          )
```

Comment:
Very nice idea and well done on passing this as a function!
Even better though, make it a separate function (not pass it directly using lambda)
and pass it just for readability reasons: